src/styles/home_nav.css

```diff
@@ -0,0 +1,213 @@
+  transition: color 0.3s ease;
+  color: #9da0a7;
+  font-size: .8em;
+  font-weight: 400;
+}
+
+.home-nav a:hover {
+  cursor: pointer;
+  color: #fff;
+}
+
+.nav-container {
+  padding-left: 40px;
+  padding-right: 40px;
+  background-color: #3d4351;
+  font-family: 'Catamaran', sans-serif;
+  font-weight: 300;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  overflow: hidden;
+}
+
+.logo{
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin: 14px 0 14px 0;
+  width: 210px;
+  height: 36px;
+}
+
+.logo-green-square {
+  background-color: #8dc63f;
+  height: 33px;
+  width: 33px;
+  min-width: 33px;
+  display: flex;
+  align-items: center;
+}
+
+.logo-green-square div {
+  font-size: 2em;
+  color: #3d4351;
+  font-weight: 200;
+  position:absolute;
+  left:46px;
+  top:9px;
+}
+
+.logo img {
+  max-height: 35px;
+}
+
+@media only screen and (max-width: 1200px) and (max-width: 1030px) {
+  .logo-type {
+    visibility: hidden;
+  }
+}
+
+.logo-type {
+  margin-left: 10px;
+  max-height: 15px !important;
+}
+
+.home-nav-right{
+  flex-grow: 1;
+}
+
+.home-nav-menu {
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: flex-end;
+  flex-direction: row;
+  align-items: center;
+  max-height: 64px;
+  overflow: hidden;
+}
+
+.home-nav-menu a {
+  margin-left: 25px;
+}
+
+.nav-dropdown span {
+  display: block;
+}
+
+@media only screen and (max-width: 720px) and (max-width: 1120px) {
+  .home-nav-right {
+    visibility: hidden;
+  }
+}
+
+.nav-dropdown {
+  display: inline-block;
+}
+
+.nav-dropdown div {
+  margin-right: 10px
+}
+
+.more-triangle {
+  border-bottom: 5px solid transparent;
+  border-left: 5px solid transparent;
+  border-right: 5px solid transparent;
+  border-top: 5px solid ;
+  max-width: 0px;
+  max-height: 0px;
+  position: absolute;
+  right: 520px;
+  top: 29px;
+  z-index: 15;
+}
+
+.home-nav-triangle {
+  border-bottom: 10px solid white;
+  border-left: 10px solid transparent;
+  border-right: 10px solid transparent;
+  border-top: 10px solid transparent;
+  max-width: 0px;
+  max-height: 0px;
+  position: absolute;
+  right: 545px;
+  top: 38px;
+  z-index: 15;
+}
+
+.home-nav-dropdown {
+  position: absolute;
+  background-color: #fff;
+  min-width: 100px;
+  z-index: 10;
+  border-radius: 3px;
+  right: 458px;
+  top: 58px;
+  box-shadow: 0 0 10px rgba( 0,0,0,.1 );
+  transition: all .2s ease-in-out;
+}
+
+.home-nav-dropdown ul {
+  z-index: 11;
+}
+.home-nav-dropdown ul li {
+  list-style: none;
+  z-index: 12;
+  text-align: center;
+  justify-content: center;
+  padding: 15px 10px;
+  font-size: .9em;
+}
+
+.home-nav-dropdown ul li a {
+  color: #888;
+  font-weight: 600;
+  padding: 0px;
+  margin: 0px;
+}
+
+.home-nav-dropdown ul li a:hover {
+  color: #333;
+}
+
+.home-nav-dropdown ul li:not( :last-child ) {
+  border-bottom: solid 1px #ccc;
+}
+
+.home-nav-dropdown-content {
+  padding: 0px;
+  margin: 0px;
+}
+
+.login-nav {
+  color: #fff !important;
```

Comment:
All these `!important` should not be needed if we've specified our styles correctly...